openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1655,6 +1655,7 @@ static int cert_requested(SSL* ssl, X509** x509Out, EVP_PKEY** pkeyOut) {
             break;
     }
 #else
+    (void) ssl2_ctype;
```

Comment:
can we instead just declare this variable in the same `ifdef` condition in which it is used in?